contracts/contracts/vault/VaultStorage.sol

```diff
@@ -89,10 +89,10 @@ contract VaultStorage is Initializable, Governable {
 
     /// @dev mapping of supported vault assets to their configuration
     // slither-disable-next-line uninitialized-state
-    mapping(address => Asset) internal assets;
+    mapping(address => Asset) public assets;
     /// @dev list of all assets supported by the vault.
     // slither-disable-next-line uninitialized-state
-    address[] internal allAssets;
+    address[] public allAssets;
```

Comment:
do we need to add this to the ABI when we already have `getAllAssets`?

Reply:
Yes, will revert this and the following changes